app/_src/policies/meshservice.md

```diff
@@ -0,0 +1,94 @@
+---
+title: MeshService
+---
+
+{% warning %}
+This resource is experimental!
+In Kubernetes, to take advantage of the automatic generation described below,
+you need to set both [control plane configuration variables](/docs/{{ page.version }}/reference/kuma-cp/) `KUMA_EXPERIMENTAL_SKIP_PERSISTED_VIPS`
+and `KUMA_EXPERIMENTAL_GENERATE_MESH_SERVICES` to `"true"` on the zone control
+planes that use `MeshServices`.
+{% endwarning %}
+
+MeshService is a new resource that represents what was previously expressed by
+the `Dataplane` tag `kuma.io/service`. Kubernetes users should think about it as
+the analog of a Kubernetes `Service`.
+
+A basic example follows to illustrate the structure:
+
+```
+type: MeshService
```

Comment:
I tried to use this yaml for the validator's test and it seems incorrect:
> Error from server (BadRequest): error when creating "STDIN": MeshService in version "v1alpha1" cannot be handled as a MeshService: strict decoding error: unknown field "spec.ports[0].targetPort.value", unknown field "spec.ports[1].targetPort.name"

Reply:
I used a slightly modified version for k8s:
```yaml
apiVersion: kuma.io/v1alpha1
kind: MeshService
metadata:
  name: redis
  labels:
    kuma.io/mesh: default
    team: db-operators
spec:
  selector:
    dataplaneTags:
      app: redis
      k8s.kuma.io/namespace: redis-system
      kuma.io/zone: east-1
  ports:
    - port: 6739
      targetPort:
        value: 6739
      appProtocol: tcp
    - name: some-port
      port: 16739
      targetPort:
        name: target-port-from-container
      appProtocol: tcp
status:
  addresses:
    - hostname: redis.mesh
      origin: HostnameGenerator
      hostnameGeneratorRef: my-hostname-generator
  vips:
    - ip: 10.0.1.1
```

Reply:
The error is in targetPort, should be:
```yaml
apiVersion: kuma.io/v1alpha1
kind: MeshService
metadata:
  name: redis
  labels:
    kuma.io/mesh: default
    team: db-operators
spec:
  selector:
    dataplaneTags:
      app: redis
      k8s.kuma.io/namespace: redis-system
      kuma.io/zone: east-1
  ports:
    - port: 6739
      targetPort: 6739
      appProtocol: tcp
    - name: some-port
      port: 16739
      targetPort: target-port-from-container
      appProtocol: tcp
```

Reply:
updated the resource